chapters/03_feed-forward.tex

```diff
@@ -297,20 +301,20 @@ \section{Conditionals}\label{sec:ffnn_conditional}
 \section{Multiplication}
 \label{sec:ffnn_multiplication}
 
-By definition~\footnote{\url{https://pytorch.org/docs/stable/generated/torch.nn.GELU.html}}:
+GELU is implemented\footnote{\url{https://pytorch.org/docs/stable/generated/torch.nn.GELU.html}} as in Equation~\eqref{eq:gelu}, hence:
 \begin{align*}
-    \mrm{GeLU}(x) \equiv \frac{x}{2} \parens*{1 + \tanh\parens*{\sqrt{\frac{2}{\pi}} (x + 0.044715x^3)}}
+    \mrm{GELU}(x) = \frac{x}{2} \parens*{1 + \tanh\parens*{\sqrt{\frac{2}{\pi}} (x + 0.044715x^3)}}
```

Comment:
This is now just a duplicate of the definition, so should it be deleted?

Reply:
I don't think so. Since that is how it is implemented and for the purpose of the construction, we state that the GELU approximation is now exact. Though, we could also just mention that when referencing the approximate definition.